CHANGELOG.md

```diff
@@ -563,4 +574,4 @@ Initial release.
 * spl: `anchor-spl` crate providing CPI clients for Anchor programs.
 * client: `anchor-client` crate providing Rust clients for Anchor programs.
 * ts: `@project-serum/anchor` package for generating TypeScript clients.
-* cli: Command line interface for managing Anchor programs.
+* cli: Command line interface for managing Anchor programs.
```

Comment:
what happened here? I saw that in a different PR too 🤔

Reply:
probably just a github bug?

Reply:
Visual Studio Code doing weird stuff when I saved the document 😵‍💫

Reply:
oh maybe a different os line break?

Reply:
Maybe, not sure, honestly, but I think it is more than just that. For example, if I use VSCode, when I save, all the `*` get replaced by  `-.`